install/kubernetes/helm/istio/values.yaml

```diff
@@ -325,3 +325,7 @@ global:
   # Use the Mesh Control Protocol (MCP) for configuring Mixer and
   # Pilot. Requires galley (`--set galley.enabled=true`).
   useMCP: false
+
+  mixer:
```

Comment:
These config options should go in:

https://github.com/istio/istio/blob/master/install/kubernetes/helm/subcharts/mixer/values.yaml

Reply:
If people want to enable/disable this setting when they deploy the core Istio chart, this is the only way to do it centrally.  Otherwise, they have to discover and manually go and modify the sub-chart values.  Is this the desired user experience?

From [the comment on the future status of this setting](https://github.com/istio/istio/issues/7675#issuecomment-432838799) it appears eventually this switch will go away and should be removed from the helm chart in Istio 1.3.0.

Reply:
@sdake I can easily change the chart to move this setting, but think it is a better user experience as it is now... 
I assume from lack of response you don't want to move forward as-is?

Reply:
@dtzar really sorry for lack of response - have been on pto, then work travel.

Reply:
The mixer subcharts are a better place to put this I think - assuming we care to install CRDs incrementally.  I personally believe that incremental CRD adoption makes this whole mess of 1.1 upgrades harder to sort out.  I'd be in favor of no conditionals per service for CRD instantiation - except where mandatory.